### ISSUE TITLE
composer install error.

### ISSUE THREAD

**Issue by hgati** [magefan/module-translation#14]
I'm using PHP 8.2 supported by Magento v2.4.6

Deprecated Functionality: Creation of dynamic property Magefan\Crowdin\Model\GetTranslationEntity::$categoryCollectionFactory is deprecated in /var/www/magento/vendor/magefan/module-crowdin-integration/Model/GetTranslationEntity.php on line 77

![](https://i.ibb.co/rMhL1TJ/2023-03-28-19-38-29.png)

![](https://i.ibb.co/0K5ctWG/2023-03-28-19-44-11.png)

[Reference] https://php.watch/versions/8.2/dynamic-properties-deprecated

**Comment by magefan** [comment]
@hgati,

We've released a new version of magefan/module-crowdin-integration 2.0.4 that has a fix for this issue, please run the command

composer update magefan/module-crowdin-integration

Let me know if it helps.

**Comment by hgati** [comment]
> @hgati,
> 
> We've released a new version of magefan/module-crowdin-integration 2.0.4 that has a fix for this issue, please run the command
> 
> composer update magefan/module-crowdin-integration
> 
> Let me know if it helps.

Another error message. 
`composer require magefan/module-crowdin-integration`
**Unique constraint violation found**

![](https://i.ibb.co/7YX67wg/magefan-trans.png)

`{"0":"Please upgrade your database: Run \"bin\/magento setup:upgrade\" from the Magento root directory.\nThe following modules are outdated:\nMagefan_Crowdin data: current version - none, required version - 2.0.0","1":"#1 Magento\\Framework\\App\\FrontController\\Interceptor->Magento\\Framework\\Interception\\{closure}() called at [vendor\/magento\/module-page-cache\/Model\/App\/FrontController\/BuiltinPlugin.php:71]\n#2 Magento\\PageCache\\Model\\App\\FrontController\\BuiltinPlugin->aroundDispatch() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:135]\n#3 Magento\\Framework\\App\\FrontController\\Interceptor->Magento\\Framework\\Interception\\{closure}() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:153]\n#4 Magento\\Framework\\App\\FrontController\\Interceptor->___callPlugins() called at [generated\/code\/Magento\/Framework\/App\/FrontController\/Interceptor.php:23]\n#5 Magento\\Framework\\App\\FrontController\\Interceptor->dispatch() called at [vendor\/magento\/framework\/App\/Http.php:116]\n#6 Magento\\Framework\\App\\Http->launch() called at [vendor\/magento\/framework\/App\/Bootstrap.php:264]\n#7 Magento\\Framework\\App\\Bootstrap->run() called at [pub\/index.php:30]\n","url":"\/","script_name":"\/index.php","report_id":"f9a07a8bfa61349187bc3f8d700855df8322f41ff9a3ad39d2b51a0c744e1d7c"}
`

so I run this command.
`magento setup:upgrade`
same error. **Unique constraint violation found**

**Comment by hgati** [comment]
![](https://i.ibb.co/19XQK63/hack.png)

After installing your extension I got this.
What is this?
And, my site is linked to the crowdin site?
Are product information and other information on my site automatically transmitted to the crowdin site?
I installed this module to manually translate only translation tables in my database. 
However, the newly informed extension has a different intention?

How can I delete this? No delete button.
I deleted the extension - magefan/module-crowdin-integration, but this information still exists and cannot be deleted.
